### PR TITLE
Changed UNSAFE method to componentDidMount and some other visual changes

### PR DESCRIPTION
**Pull request by FrissA**
Changed UNSAFE method to componentDidMount and some other few visual changes. These solved the warnings that popped out when using the library by itself.

### CODE REVIEW

**Comment by oyemeLife** [comment]
> Changed UNSAFE method to componentDidMount and some other few visual changes. These solved the warnings that popped out when using the library by itself.

Can't wait for changes.

File: DrawerLayout.js [hunk 1]
```diff
@@ -450,8 +450,7 @@ export default class DrawerLayout extends Component<PropType, StateType> {
               : styles.containerInFront,
             containerStyles,
             contentContainerStyle,
-          ]}
-          importantForAccessibility={this._drawerShown ? "no-hide-descendants" : "yes"}>
```
Comment (osdnk) [inline DrawerLayout.js]: Why have you dropped this?

**Comment by jakub-gonet** [comment]
Hi @FrissA, any update on this?

**Comment by jakub-gonet** [comment]
Closing due to no response.

**Pull request closed without merging**